src/main/java/io/netty/incubator/channel/uring/IOUringEventLoop.java

```diff
@@ -70,6 +76,21 @@
         logger.trace("New EventLoop: {}", this.toString());
     }
 
+    /**
+     * Submit the IO so the kernel can process it. This method can be called to "force" the submission (before it
+     * is submitted by netty itself).
+     */
+    public void submitIO() {
+        if (inEventLoop()) {
+            if (getRingBuffer().ioUringSubmissionQueue().submit() > 0 &&
+                    getRingBuffer().ioUringCompletionQueue().hasCompletions()) {
+                //getRingBuffer().ioUringCompletionQueue().process(callback);
```

Comment:
Remove commented out code, or un-comment it back in?

Reply:
doh... done